### DISCUSSION TITLE
How can I perform 3D reconstruction using large image datasets?

### DISCUSSION THREAD

**Discussion by kbhyun12** [astra-toolbox/astra-toolbox#487] [Q&A]
I captured 1440 X-Ray images of size 1800x1800 while rotating the object 360 degrees. What algorithm is suitable for 3D reconstruction from these images? Also, is it possible to perform the reconstruction quickly?

**Comment by askorikov** [comment]
Hi! FDK (for cone beam geometry) or slice-by-slice FBP (for parallel geometry) algorithm is your best bet.

**Reply by kbhyun12** [reply]
Thank you for your response.

According to the manual documentation, FDK pertains to 3D data, while FBP pertains to 2D data.
Which algorithm would be more suitable between 3D and 2D?

Could you please provide a more detailed explanation regarding the application of FDK and FBP algorithms?

**Reply by askorikov** [reply]
Indeed, FDK is for 3D (cone) geometry and FBP is for 2D geometries. However, you can apply FBP slice-by-slice in 3D if your geometry is parallel.
